apps/bookmark/js/bookmark_editor.js

```diff
@@ -54,7 +70,24 @@ var BookmarkEditor = {
 
   _renderIcon: function renderIcon() {
     var icon = new Icon(this.bookmarkIcon, this.data.icon);
-    icon.render();
+    icon.render({'size': this.BOOKMARK_ICON_SIZE});
+  },
+  
+  _getManifest: function bookmarkEditor_getManifest(manifestURL) {
+    var manifestPromise = window.WebManifestHelper.getManifest(manifestURL);
+    
+    manifestPromise.then((function(manifestData) {
+      if (manifestData) {
+        this.appInstallationSection.classList.remove('hidden');
+        this.manifest = manifestData;
```

Comment:
We could save a reference to manifest like 

```js
var manifest = this.manifest = manifestData;
```

to use in next line although if you consider this interesting